Rapport open ruimte kaart (#1)
inbo/patrijs-steekproefkader: rapport/index.Rmd

    email: "thierry.onkelinx@inbo.be"
    orcid: 0000-0002-1825-0097
  - firstname: Raïsa
    name: Carmen
    email: "raisa.carmen@inbo.be"
    orcid: 0000-0003-1025-8702
style: INBO
lang: nl

reviewer: 
  - firstname: Thomas
    name: Scheppers
    email: "thomas.scheppers@inbo.be"
    orcid: 0000-0002-5098-9326
year: 2022
reportnr: 2
depotnr: D/2022/3241/023
doi: doi.org/10.21436/inbor.70809860

# report specific options. See ?INBOmd::report() for full list and more details.
shortauthor: Onkelinx, T. & Carmen, R.
corresponding: thierry.onkelinx@inbo.be
cover_photo: https://img5.vildaphoto.net/asset/l/117545.jpg
cover_description: Roepende patrijs in maïsveld. © Yves Adams

# pdf report specific options. See ?INBOmd::report() for full list and more details.
embargo: 2021-12-10

# HTML report specific options. See ?INBOmd::gitbook() for full list and more details.
github-repo: inbo/patrijs-steekproefkader

floatbarrier: section

bibliography: references.bib
link-citations: TRUE
site: bookdown::bookdown_site

output:
  INBOmd::gitbook: default
  INBOmd::report: default
  INBOmd::ebook: default
  bookdown::dontrun: default
---

# Samenvatting {-}

**Waarom een open ruimte kaart?**

> Het Agentschap voor Natuur en Bos (ANB) opent de jacht op patrijs een gebied enkel indien aan meerdere voorwaarden voldaan is.

Reviewer: ```suggestion
Het Agentschap voor Natuur en Bos (ANB) opent de jacht op patrijs in een gebied enkel indien aan meerdere voorwaarden voldaan is.
```

> Het Agentschap voor Natuur en Bos (ANB) opent de jacht op patrijs een gebied enkel indien aan meerdere voorwaarden voldaan is.

Reviewer 2: ```suggestion
Het Agentschap voor Natuur en Bos (ANB) opent de jacht op patrijs in een wildbeheereenheid enkel indien aan meerdere voorwaarden voldaan is.
```

> Een van deze voorwaarden is dat er voldoende patrijzen aanwezig zijn.
> Hiervoor gebruikt ANB als dat er minstens 3 broedparen per 100 ha open ruimte zijn.

Reviewer: ```suggestion
Een van deze voorwaarden is dat er voldoende patrijzen aanwezig zijn.
ANB hanteert een minimum van 3 broedparen per 100 ha open ruimte.
```

> Een van deze voorwaarden is dat er voldoende patrijzen aanwezig zijn.
> Hiervoor gebruikt ANB als dat er minstens 3 broedparen per 100 ha open ruimte zijn.

Reviewer: ```suggestion
Een van deze voorwaarden is dat er voldoende patrijzen aanwezig zijn.
ANB hanteert een minimum van 3 broedparen per 100 ha open ruimte.
```

Daarom moet een wildbeheereenheid (WBE) die op patrijs wenst te jagen, eerst inventariseren waar er allemaal patrijzen rondlopen.
Op basis van deze waarnemingen bepaalt het Instituut voor Natuur- en Bosonderzoek (INBO) het aantal broedparen in elke WBE.
Tenslotte delen we dit aantal broedparen door de oppervlakte open ruimte om tot de dichtheid van patrijs (broedparen per 100 ha open ruimte) te komen.
Om deze oppervlakte te kennen, hebben we nood aan een kaart die bepaalt wat open ruimte is.

**Waarom een nieuwe open ruimte kaart?**

> Ne de tellingen van voorjaar 2021 lieten meerdere jagers ons weten dat de gebruikte open ruimte kaart niet altijd overeenkomt met de werkelijkheid.

Reviewer: ```suggestion
Na de tellingen van voorjaar 2021 lieten meerdere jagers ons weten dat de gebruikte open ruimte kaart niet altijd overeenkomt met de werkelijkheid.
```

> Ne de tellingen van voorjaar 2021 lieten meerdere jagers ons weten dat de gebruikte open ruimte kaart niet altijd overeenkomt met de werkelijkheid.

Reviewer 2: ```suggestion
Na de tellingen van het voorjaar 2021 lieten meerdere jagers ons weten dat de gebruikte open ruimte kaart niet altijd overeenkomt met de werkelijkheid.
```

> Sommige percelen zijn open ruimte maar bebouwd of bebost.

Reviewer 2: ```suggestion
Sommige percelen zijn aangeduid als open ruimte maar zijn in werkelijkheid bebouwd of bebost.
```

Waardoor de oppervlakte open ruimte overschat en de dichtheid van patrijs onderschat is.

De vorige versie van de open ruimte kaart is gebaseerd op de meest recente publicatie van de biologische waarderingskaart (BWK).

> Hoewel deze versie in 2020 gepubliceerd is, wil dat niet zeggen dat alle informatie uit 2020 stamt.

Reviewer 2: ```suggestion
Hoewel deze versie in 2020 gepubliceerd werd, wil dat niet zeggen dat alle informatie uit 2020 stamt.
```

> In de praktijk bleek zowat driekwart ouder dan 2007.

Reviewer 2: ```suggestion
In de praktijk bleek zowat driekwart van de gegevens ouder dan 2007.
```

Vandaar dat we op zoek gaan naar een betere bron.

**Een _open_ open ruimte kaart**

> Na het afwegen van een aantal mogelijkheden, kozen we [OpenStreetMap](https://www.openstreetmap.org) als bronkaart.

Reviewer 2: ```suggestion
Na het afwegen van een aantal mogelijkheden, kozen we [OpenStreetMap](https://www.openstreetmap.org) als nieuwe bronkaart.
```

Net zoals alle andere kandidaten, is deze kaart niet perfect.
Ze is evenwel goed genoeg om te gebruiken.
En heeft als troef dat fouten relatief eenvoudig en snel te verbeteren zijn.
Bij de start van de inventarisatie krijgen de jagers een voorlopige versie van de open ruimte kaart.
Wanneer een jager een fout in de open ruimte kaart vaststelt, kan deze de fout melden of zelf aanpassen in OpenStreetMap.

> Na het berekenen van het aantal broedparen kunnen we de finale versie van de open ruimte kaart op basis van de meest recente informatie uit OpenStreetMap.

Reviewer: ```suggestion
Na het berekenen van het aantal broedparen kunnen we de finale versie van de open ruimte kaart vastleggen op basis van de meest recente informatie uit OpenStreetMap.
```

Dit is een belangrijk verschil met de overige bruikbare kaarten.
Deze beschikken niet over een mogelijkheid om ze bij te werken.

> Verder zijn ze gebaseerd om informatie die minstens 1 tot 2 jaar oud is.

Reviewer: ```suggestion
Verder zijn ze gebaseerd op informatie die minstens 1 tot 2 jaar oud is.
```

Voor de jagers biedt deze manier van werken hun de mogelijkheid om zelf bij te dragen aan een correcte open ruimte kaart.
Aangezien OpenStreetMap de informatie met open gebruiksvoorwaarden aanbiedt, kan iedereen genieten van een verbeterde OpenStreetMap kaart.

**Een [reproduceerbare](https://nl.wikipedia.org/wiki/Reproduceerbaarheid) open ruimte kaart**

In dit rapport beschrijven we hoe we de informatie van OpenStreetMap omzetten naar een open ruimte kaart.
De omzetting hebben we als een computerprogramma uitgewerkt.

> We publiceren die computerprogramma samen met de open ruimte kaart en de gebruikte bronkaarten (OpenStreetMap en grenzen jachtgebieden).

Reviewer: ```suggestion
We publiceren dit computerprogramma samen met de open ruimte kaart en de gebruikte bronkaarten (OpenStreetMap en grenzen jachtgebieden).
```

Op die manier kan iedereen de gebruikte regels (laten) nakijken of zelf uitvoeren.

\benglish

# English abstract {-}

> Insert a translation of the summary here.

Reviewer: **Why do we need an open space map?**

The Agency for Nature and Forests (ANB) only allows the hunt for partridges in an area if several conditions are met.
One of these conditions is that there are enough partridges present.
For this, ANB applies a minimum of 3 breeding pairs per 100 ha of open space.
Therefore, a wildlife management unit (WMU) that wishes to hunt partridge must first make an inventory of where all the partridges are.
Based on these observations, the Research Institute for Nature and Forest (INBO) determines the number of breeding pairs in each WMU.
Finally we divide this number of breeding pairs by the area of open space to arrive at the density of partridge (number of breeding pairs per 100 ha open space).
To obtain the area of open space in each WMU, we need a map that defines what open space is.

**Why a new open space map?

After the spring 2021 censuses, several hunters informed us that the open space map that was used does not always correspond to reality.
Some plots were classified as open space on the map but were but cultivated or forested in reality.
This overestimated the area of open space and underestimated the density of partridge.
The previous version of the open space map is based on the most recent publication of the biological valuation map (BWK).
Although this version of BWK was published in 2020, that does not mean that all the information was from 2020.
actually, about three quarters of the information turned out to be older than 2007.
Hence, we are looking for a better source.

**An _open_ open space map**

After weighing a number of options, we chose [OpenStreetMap](https://www.openstreetmap.org) as our source map.
Like all the other candidates, this map is not perfect.
However, it is good enough to use.
And it has the advantage that errors are relatively easy and quick to correct.
At the start of the surveys, hunters are given a preliminary version of the open space map.
When a hunter identifies an error in the open space map, they can report the error or modify it themselves in OpenStreetMap.
After calculating the number of breeding pairs, we can provide the final version of the open space map based on the most recent information from OpenStreetMap.
This is an important difference between OpenStreetMap and other usable maps.
The others do not have an option to update them.
Furthermore, they are based on information that is at least 1 to 2 years old.

For hunters, this way of working gives them the opportunity to contribute to a correct open space map themselves.
Since OpenStreetMap provides the information with open use terms, everyone can enjoy an improved OpenStreetMap.

**A [reproducible](https://nl.wikipedia.org/wiki/Reproduceerbaarheid) open space map**

In this report, we describe how to convert the information from OpenStreetMap to an open space map.
We established the conversion as a computer program.
We publish that computer program together with the open space map and the source maps that were used (OpenStreetMap and hunting area boundaries).
In this way, everyone can check the rules (or have them checked), use, or implement them themselves.

Reviewer 2: wildlife managment unit --> game management unit (GMU)

Dat is de Engelse benaming. Dus overal GMU van maken.

Reviewer 2: "ome plots were classified as open space on the map but were but cultivated or forested in reality."

--> cultivated klopt niet. Dat moet bebouwd zijn.

Reviewer 2: " that all the information was from 2020.
actually, about three quarters of the information turned out to be older "

--> hoofdletter Actually

Reviewer 2: "When a hunter identifies an error in the open space map, they can report the error or modify it themselves in OpenStreetMap."

--> he can... himselve...

Reviewer 2: "we can provide the final version of th"

--> we can obtain

Author: Toegevoegd in feef657

\eenglish

<!-- hier komt de inhoudstafel bij pdf -->
\mainmatter

# Inleiding

Sinds 2008 is de jacht op patrijs in Vlaanderen gebonden aan een aantal voorwaarden.
De eerste voorwaarde vermeldt het voldoen aan een gemiddelde dichtheid van minstens 3 broedparen per 100 ha open ruimte.
Het Instituut voor Natuur- en Bosonderzoek (INBO) staat in voor de berekening van deze dichtheden en brengt hierover advies uit aan het Agentschap voor Natuur en Bos (ANB).

> Aangezien de jachtwetgeving geen definitie geeft van de open ruimte werd, kozen we bij de invoering van deze regelgeving om te werken met de biologische waarderingskaart ([BWK](https://www.vlaanderen.be/inbo/de-biologische-waarderingskaart/bwk-belang/)).

Reviewer: ```suggestion
Aangezien de jachtwetgeving geen definitie geeft van de open ruimte, kozen we bij de invoering van deze regelgeving om te werken met de biologische waarderingskaart ([BWK](https://www.vlaanderen.be/inbo/de-biologische-waarderingskaart/bwk-belang/)).
```

De BWK is een gebiedsdekkende inventaris van het biologische milieu en van de bodembedekking van het volledige Vlaams Gewest.
Deze kaart wordt door het INBO opgemaakt aan de hand van een inventarisatie met behulp van vooraf gedefinieerde karteringseenheden.
De verschillende karteringseenheden werden beoordeeld als behorende tot de categorie "open ruimte" of "niet-open ruimte" voor patrijs.
Op deze manier kon een afbakening van de open ruimte bepaald worden.

Voor de berekeningen van de dichtheden van de broedparen werd steeds gewerkt met de meest recente versie van de BWK.
Ook @Onkelinx2021 gebruikte de meest recente versie van de BWK [@BWK].
De verschillende versies verwijzen echter naar de kaart als geheel, niet naar de laatste update van elk polygoon in de kaart.

> Zo stelt De Saeger et al. (2020) dat in de meest actuele versie ongeveer driekwart van de oppervlakte van het niet urbaan gebied gebaseerd is op een kartering van 2007 of ouder (fig. 1‐1 in @BWK).

Reviewer 2: ```suggestion
Zo stelt De Saeger _et al._ (2020) dat in de meest actuele versie ongeveer driekwart van de oppervlakte van het niet urbaan gebied gebaseerd is op een kartering van 2007 of ouder (fig. 1‐1 in @BWK).
```

Het gevolg is dat de BWK voor sommige gebieden niet langer het actuele landgebruik weergeeft.
Wat op zijn beurt weer aanleiding geeft tot fouten in de kaart met open ruimte relevant voor patrijs.

In het kader van de uitrol van een standaard telprotocol voor patrijs, werd daarom beslist om een alternatieve bepaling van de open ruimte voor patrijs uit te werken.
Gezien de hoeveelheid vragen en opmerkingen over de vorige kaart, willen we de nieuwe kaart tevens transparant, open en reproduceerbaar maken.
Elke betrokkene kan hierdoor de werkwijze (laten) nakijken of zelf opnieuw (laten) uitvoeren.

> Om dit te kunnen garanderen gebruiken we enkel kaartmateriaal dat als [open data](https://nl.wikipedia.org/wiki/Open_data) beschikbaar is en [vrije software](https://nl.wikipedia.org/wiki/Vrije_software_en_opensourcesoftware).

Reviewer: ```suggestion
Om dit te kunnen garanderen, gebruiken we enkel kaartmateriaal dat als [open data](https://nl.wikipedia.org/wiki/Open_data) beschikbaar is en [vrije software](https://nl.wikipedia.org/wiki/Vrije_software_en_opensourcesoftware).
```

> In dit rapport beschreven we welke alternatieven kaarten we in overweging genomen hebben.

Reviewer: ```suggestion
In dit rapport beschreven we welke alternatieve kaarten we in overweging genomen hebben.
```
